jujugui/static/gui/src/app/components/budget-table/row/row.js

```diff
@@ -409,7 +393,9 @@ class BudgetTableRow extends React.Component {
           <div>
             {this._generateSharedFields()}
             {this._generatePlanCols()}
-            {this._generateExtra()}
+            <div className="twelve-col no-margin-bottom">
```

Comment:
Should this not be in a separate `this._generateExtras()` or at least in an if so we don't have empty layout divs?